### PR TITLE
DIT-1277

### PR DESCRIPTION
**Pull request by CanOfBees**
Address [DIT-1277](https://jirautk.atlassian.net/browse/DIT-1277)

This PR adds a simple identity transform stylesheet, creates two tmpl files, and serializes _uncommented_ config files.

### CODE REVIEW

File: solr-conf/solrconfig.xml [hunk 1]
```diff
@@ -1,1804 +1,397 @@
...
-      <str name="spellcheck.maxCollations">5</str>         
-    </lst>
-    <arr name="last-components">
-      <str>spellcheck</str>
-    </arr>
-  </requestHandler>
-
-  <!-- Term Vector Component
-
-       http://wiki.apa
```
Comment (DonRichards) [inline solr-conf/solrconfig.xml]: Is this correct?

**Comment by CanOfBees** [comment]
@DonRichards not sure I follow -- is what correct?

Edit: oh wait, maybe I do understand: are you asking about the diff highlighting? It does look like pieces of the file are invalid in the diff. If you look at the full file, it shows that section is still present (without comments).

Is that what you were asking about? 
Thanks!

File: solr-conf/solrconfig.xml (hunk 1 above)
Comment (CanOfBees) [reply solr-conf/solrconfig.xml]: Ugh, sorry - just realized that I responded in the wrong spot.

If I'm understanding your question, you're asking about the validity of the file based on the diff view? If that's the case, AFAICT, the diff view loads in a bit lower on the page.

Apologies for the confusion. Let me know if I'm still misunderstanding.

Comment (DonRichards) [reply solr-conf/solrconfig.xml]: Screen Shot 2019-11-06 at 2 53 02 PM
I hope this helps. 
```xml
# What is in the PR
         <str name="hl.bs.chars">.,!? 
&#xD;</str>

# What I'd expect it to be. 
         <str name="hl.bs.chars">.,!?</str> 
```

Comment (DonRichards) [reply solr-conf/solrconfig.xml]: Looked like an accidental newline with a carriage return.

Comment (CanOfBees) [reply solr-conf/solrconfig.xml]: Okay, thanks for being more specific. In the source document we have
`<str name="hl.bs.chars">.,!? &#9;&#10;&#13;</str>`. It looks like the transform stripped the entities. I'll add them back.

**Comment by markpbaggett** [comment]
@CanOfBees, are we assuming that all reviewers must review prior to approval and merge?  That's not what we did on the other pull requests, but I was wondering if this one was an exception.

I'm also not sure on what was decided here.  After rereading this numerous times, I think I understand what's being discussed, but I don't see this as being an actual technical issue.  As far as well-formedness goes, this isn't a problem, and thus should not be problematic for our processor.

**Comment by CanOfBees** [comment]
@markpbaggett - looking over recent documentation for highlighting configuration, it isn't 100% clear to me if the requirements for `hl.bs.chars` expect entities for certain chars, or is okay with the actual chars. 

I agree that it isn't a well-formed-ness problem. Despite that, I think that keeping the currently serialized values is probably best - unless we can verify that the other, actual-serialized characters still works correctly with highlighting.

Additionally, I don't think that all reviewers need to approve for a merge. If you've reviewed and are comfortable with the current status, please accept and merge. 

Does that help?
Thanks!

**Review by markpbaggett — APPROVED** [review]
I'm not sure about the problems, what's being asked, or what has been decided.  For that reason, I approve whatever.

**Comment by DonRichards** [comment]
My requests was resolved. I only noted a character irregularity that was corrected. Although I haven't tested this, it does look good to move forwards with a test & merge.

**Comment by DonRichards** [comment]
@CanOfBees Is this still needed? If so there is a merge conflict.

**Comment by CanOfBees** [comment]
@DonRichards I'm not sure, to be honest. If you want the config file to be commentless, then yes & the merge conflict will need to be addressed. If you're not concerned about comments in the config file, then we can close?

What do you think?

**Comment by DonRichards** [comment]
If there isn't a need I say we close this for now.

**Comment by CanOfBees** [comment]
Okay, I'll close this for now.

**Pull request closed without merging**